src/prop-types/historyPush.js

```diff
@@ -0,0 +1,5 @@
+import PropTypes from 'prop-types';
```

Comment:
we can call this propTypes an file as history. And always add here the things in future, if needed,